src/atom.coffee

```diff
@@ -321,18 +324,16 @@ class Atom extends Model
 
   # Extended: Invoke the given callback whenever there is an unhandled error.
   #
-  # * `callback` {Function} to be called whenever there is an unhandled error
-  #   * `event` {Object}
-  #     * `originalError` {Object} the original error object
```

Comment:
Are these fields no longer available on the object passed to the callback? Just curious why they are removed from the docs since it looks like they are still there in the emitted object.